Libraries/Text/Text.js

```diff
@@ -113,10 +152,21 @@ const Text = React.createClass({
      */
     testID: React.PropTypes.string,
     /**
-     * Specifies should fonts scale to respect Text Size accessibility setting on iOS.
+     * Specifies whether fonts should scale to respect Text Size accessibility setting on iOS. The
+     * default is `true`.
+     *
      * @platform ios
      */
     allowFontScaling: React.PropTypes.bool,
+    /**
+     * When set to `true`, indicates that the view is an accessibility element. The default value
+     * for a `Text` element is `true`.
+     *
+     * See the
+     * [Accessibility guide](/react-native/docs/accessibility.html#accessible-ios-android)
+     * for more information.
+     */
+     accessible: React.PropTypes.bool,
```

Comment:
@bestander @mkonicek This is the specific code addition in question. It was being added to the documentation via the `getDefaultProps` method below, but was never really officially declared as a prop.